packages/jobs/lib/execution/sync.ts

```diff
@@ -156,6 +158,28 @@ export async function startSync(task: TaskSync, startScriptFn = startScript): Pr
             void logCtx.debug(`Last sync date is ${lastSyncDate?.toISOString()}`);
         }
 
+        // tracking active schedule time
+        // by pushing the frequency of the sync as a metric
+        // that can be sumed up to get the total active
+        // time of the sync for a given period/account
+        const sync = await getSync({ connectionId: task.connection.id, name: task.syncName, variant: task.syncVariant });
```

Comment:
not a huge fan of making an extra db query. if you have a better solution lmk

Reply:
can you make the orchestrator return the frequency in task instead?